### PR TITLE
Add Select platform description for SleepIQ integration

### PR DESCRIPTION
**Pull request by mfugate1**
## Proposed change

Add documentation for the select entity added to the SleepIQ integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68489
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
`ha_plaforms` and `ha_category` needs to be updated too

File: source/_integrations/sleepiq.markdown [hunk 1]
```diff
@@ -17,6 +22,8 @@ ha_platforms:
   - binary_sensor
   - button
   - light
+  - number
```
Comment (frenck) [inline source/_integrations/sleepiq.markdown:25]: The number entity is not added in the linked PR, thus this change seems unrelated?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/sleepiq.markdown (hunk 1 above)
Comment (mfugate1) [reply source/_integrations/sleepiq.markdown:25]: I noticed a couple missing categories/platforms so I figured I'd add them all at once (number was added earlier this month). I can do that in a separate PR if needed

Comment (mfugate1) [reply source/_integrations/sleepiq.markdown:25]: I can do it here instead: https://github.com/home-assistant/home-assistant.io/pull/21925

**Review by frenck — APPROVED** [review]
